### ISSUE TITLE
Teradata 15.00: Errors in mapping of "struct SLOB_data_struct  *using_SLOB_data_struct;" to Python struct for dbcarea.py

### ISSUE THREAD

**Issue by fivejjs** [xlfe/dwhwrapper#1]
Possible reasons: 
1. type define was done with two lines:
"  char fill_20[1];
    struct SLOB_data_struct
            *using_SLOB_data_struct;
    char fill_21[376];"
1. the mapping are not working for 'struct' type of C language.

**Comment by xlfe** [comment]
Looks like the bteq api has changed. If you feel so inclined you could modifying the dwhwrapper code to ignore that structure or using teradata utils version 13

**Comment by mikekilpatrick** [comment]
I ran into some similar issues with some data types in TD 14.0 that were not supported.  I wound up downloading the teradata documents regarding supported types and added them as new types to the dwhwrapper code.  I should do that an update for the current release of TD.  Are we able to merge changes into the source tree here and make an update available to the community?

**Comment by xlfe** [comment]
@mikekilpatrick yes! That would be great - thanks for sharing your updates. Please submit a pull request with your changes.
